pkg/infrastructure/services/deployer_resources.go

```diff
@@ -244,16 +237,17 @@ func (s *serviceDeployer) getComparator() compare.MapComparator {
 	return compare.MapComparator{Comparator: resourceComparator}
 }
 
-func (s *serviceDeployer) fetchKogitoInfraProperties() (map[string]string, []corev1.EnvVar, error) {
+func (s *serviceDeployer) fetchKogitoInfraProperties() (map[string]string, []corev1.EnvVar, []v1alpha1.KogitoInfraVolume, error) {
```

Comment:
wouldn`t be better break this func into smaller ones? Or every place this function is called, all returned parameters are used?

Reply:
This function is called once, and we do only one iteraction in the `KogitoInfra`s linked to the service. Breaking it would make us to have thrice iteractions.